srcpkgs/lyx/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'lyx'
 pkgname=lyx
-version=2.3.6.1
+version=2.3.7
+_revision=1
```

Comment:
Maybe we should encode the revision in the version string as `r1`, then it more directly relates to the original download package and we can drop `_revision` in favor of a substitution in `distfiles` below:
```suggestion
version=2.3.7r1
```